lua/orgmode/agenda/init.lua

```diff
@@ -306,17 +306,30 @@ function Agenda:search(clear_search)
 end
 
 -- TODO: Add PROP/TODO Query
-function Agenda:tags(clear_search)
-  if clear_search then
+function Agenda:tags(opts)
+  local tags
```

Comment:
Move opts default up and define tags as default from opt.tags:

```
opts = opts or {}
local tags = opts.tags

if not tags then
 tags = vim.fn.input('Match: ', self.last_search, 'customlist,v:lua.orgmode.autocomplete_agenda_filter_tags')
endif
```

Reply:
Done!